src/rebar_dir.erl

```diff
@@ -133,16 +133,22 @@ home_dir() ->
     {ok, [[Home]]} = init:get_argument(home),
     Home.
 
+basedir(Type) ->
+    filename:basedir(Type, rebar3, #{os => linux}).
```

Comment:
Not sure, but this will be working on `linux`, what about `windows` and `darwin`? Or it is expected that this functionality will not be used on other OS?

Reply:
Maybe better to use `os.type/0` to detect OS and use it.

Reply:
It will work on all OSes. It just forces the uniform behaviour through all OSes. So it will work everywhere, just will follow XDG specs instead of given platform "best practices".